webapp/portlet/src/main/webapp/vue-apps/activity-reactions/components/ActivityReactionsDrawer.vue

```diff
@@ -142,31 +97,29 @@ export default {
       this.drawerOpened = true;
       if (this.lastLoadedActivityId !== this.activityId) {
         this.limit = 10;
-        this.likers = [];
         this.lastLoadedActivityId = this.activityId;
-        this.retrieveLikers();
       }
     },
     loadMore() {
       this.limit += 10;
-      this.retrieveLikers();
-    },
-    retrieveLikers() {
-      this.$refs.activityReactionsDrawer.startLoading();
-      this.$activityService.getActivityLikers(this.activityId, 0, this.limit)
-        .then(data => this.likers = data && data.likes || [])
-        .finally(() => this.$refs.activityReactionsDrawer.endLoading());
     },
     cancel() {
       this.$refs.activityReactionsDrawer.close();
     },
     refreshReactions() {
       this.activityReactionsExtensions= [];
-      const contentsToLoad = extensionRegistry.loadExtensions('activity-reactions', 'activity-reactions') || [];
+      const componentsToLoad = extensionRegistry.loadComponents('ActivityReactions') || [];
       // eslint-disable-next-line eqeqeq
-      this.activityReactionsExtensions = contentsToLoad.filter(contentDetail => contentDetail.activityId == this.activityId );
-      this.$emit('reactions', this.kudosNumber);
+      this.activityReactionsExtensions = componentsToLoad;
```

Comment:
Remove the useless declared const `componentsToLoad`.
```suggestion
      this.activityReactionsExtensions = extensionRegistry.loadComponents('ActivityReactions') || [];
```